testing/resources/template/header

```diff
@@ -0,0 +1,18 @@
+%% md
+# Welcome to PerftestNotebook
+
+press the :fast_forward: button on your top left corner to run whole notebook
+
+%% fetch
+
+text: data_string = http://127.0.0.1:5000/data
+
+%% js
+
+var data_object = JSON.parse(data_string);
+
+
```

Comment:
You don't need to have anything below this line.

Reply:
ok I see what you mean. but we still need the "from js import data_object" to pass that data from js to python.